Source/PeterKottas.DotNetCore.WindowsService/ServiceRunner.cs

```diff
@@ -304,7 +320,6 @@ private static void ConfigureService(HostConfiguration<SERVICE> config)
                     serviceHost.Run();
                     break;
                 case ActionEnum.RunInteractive:
-                    Start(config);
```

Comment:
Why do we want this gone? Just checking if it won't break some of functionality people are already using.

Reply:
That line is replaced by lines 141 - 158 in ServiceRunner.cs. I moved the actual execute into ServiceRunner.Run so the new ConsoleServiceHost result is returned as the result of ServiceRunner.Run